src/message_interface/message.rs

```diff
@@ -117,6 +120,23 @@ pub enum Message {
     /// Updates the client options for all accounts.
     /// Expected response: [`Ok`](crate::message_interface::Response::Ok)
     SetClientOptions(Box<ClientOptions>),
+    /// Generate an address without storing it
+    /// Expected response: [`Bech32Address`](crate::message_interface::Response::Bech32Address)
+    GenerateAddress {
+        /// Account index
+        #[serde(rename = "accountIndex")]
+        account_index: u32,
+        /// Internal address
+        internal: bool,
+        /// Account index
+        #[serde(rename = "addressIndex")]
+        address_index: u32,
+        /// Options
+        options: Option<GenerateAddressOptions>,
```

Comment:
What constitutes an option btw ? Would bech32 qualify as an option or ... ? I have no opinion, just asking

Reply:
Not sure :trollface: 
I also thought about adding the bech32 hrp there, but `GenerateAddressOptions` is used in iota.rs in places where we return the address not bech32 encoded, so a bech32 hrp doesn't make sense there
Then we could do nested options, but I think having it just separated is better than that

Reply:
Not sure :trollface: 
I also thought about adding the bech32 hrp there, but `GenerateAddressOptions` is used in iota.rs in places where we return the address not bech32 encoded, so a bech32 hrp doesn't make sense there
Then we could do nested options, but I think having it just separated is better than that